stack.yaml

```diff
@@ -1,12 +1,12 @@
-resolver: lts-6.7
+resolver: lts-7.4
 extra-deps:
   - Ranged-sets-0.3.0
-  - bytestring-tree-builder-0.2.7
-  - hasql-0.19.12
+  - bytestring-tree-builder-0.2.7.1
```

Comment:
we don't need this line, bytestring-tree-builder-0.2.7.1 is included in LTS 7.4
Other packages that could be removed from extra-deps because they are included are:
* hasql-0.19.15.2
* jwt-0.7.2
* postgresql-binary-0.9.1.1
* binary-parser-0.5.2
* placeholders-0.1